src/Tizen.NUI/src/devel/Common/DevelFrameUpdateCallbackInterface.cs

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright(c) 2025 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.ComponentModel;
+
+namespace Tizen.NUI
+{
+    public partial class FrameUpdateCallbackInterface
+    {
+        /// This will be public opened after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected bool GetPosition(uint id, out UIVector2 position)
+        {
+            float x = 0.0f;
+            float y = 0.0f;
+            bool ret = false;
+            if (proxyIntPtr != IntPtr.Zero)
+            {
+                ret = Interop.FrameUpdateCallbackInterface.FrameCallbackInterfaceGetPositionVector2Componentwise(proxyIntPtr, id, ref x, ref y);
+            }
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+            position = new UIVector2(x, y);
+            return ret;
+        }
+
+        /// This will be public opened after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected bool BakePosition(uint id, UIVector2 position)
+        {
+            if (proxyIntPtr == IntPtr.Zero)
+            {
+                return false;
+            }
+            bool ret = Interop.FrameUpdateCallbackInterface.FrameCallbackInterfaceBakePositionVector2Componentwise(proxyIntPtr, id, position.X, position.Y);
```

Comment:
FYI, there is a `ReusablePool<T>` that reuses object for PInvoke.
For example,
https://github.com/Samsung/TizenFX/blob/9258291d1f5257a8916399c5c8071a41d946daea/src/Tizen.NUI/src/internal/Common/Object.cs#L331-L344
Thic code get `Vector4` object from the pool and use it for PInvoke and return it back to the pool.
By using this, you can avoid to create bindings for every struct cases.

Reply:
As I know, `ReusablePool` is not thread safe. This API will be called at RenderThread, so I think we cannot use this pool here :(

Reply:
Oh my.. It need to be thread safe! since current resuable pool is not thread safe you would not use it.